extensions/bitwarden/src/utils/identity.ts

```diff
@@ -15,6 +15,7 @@ ${Object.entries(identity)
 
     return `| ${shortcutKey ? `${shortcutKey}.` : "&nbsp;"}\t**${label}** | ${value} |`;
   })
+  .filter((ele) => ele !== null)
```

Comment:
That's definitely missing 😅 

You can filter by any falsy actually.

Could you also make this change in the `utils/cards.ts` file? Kinda the same login and also has this bug.

```suggestion
  .filter(Boolean)
```

Reply:
Sure will do that right away!